src_c/scrap.c

```diff
@@ -113,10 +113,6 @@ _scrap_init(PyObject *self, PyObject *args)
     /* In case we've got not video surface, we won't initialize
      * anything.
      */
-#if IS_SDLv1
-    if (!SDL_GetVideoSurface())
-        return RAISE(pgExc_SDLError, "No display mode is set");
-#endif
```

Comment:
The thing with this is that we actually would need this check if we used the old windows scrap code, which we might do in the future, to get image support back. Because the windows scrap code needs a window handle iirc.

Also the comment is still there.

I say leave this code but have it commented out?

Reply:
Hmm, aight!